### PR TITLE
adding some Gemnasium info

### PR DESCRIPTION
**Pull request by wslack**
This clarifies some points of confusion for me when setting up Gemnasium

### CODE REVIEW

File: _pages/security/static-analysis.md [hunk 1]
```diff
@@ -21,14 +21,19 @@ Service | Dependency analysis | Code security analysis
...
 
+Gemnasium is used to scan your code for possible security issues and provides alerts for new issues that come to light.
+
 ##### Setting up your account
 
-1. [Sign into Gemnasium via GitHub.](https://gemnasium.com/login)
+1. [Sign into Gemnasium using your GitHub account.](https://gemnasium.com/login)
 1. Go to [your settings page](https://gemnasium.com/settings) and change it to use your work email, if it isn't already.
 1. Ask in [#infrastructure](https://gsa-tts.slack.com/messages/infrastructure/) to be invited to the devops@gsa.gov account.
+1. This provides you with access to the common dashboard for our projects. Make sure that your project also has a backup with access to the dashboard.
```
Comment (hollyallen) [inline _pages/security/static-analysis.md]: Consider changing the language around "backup" to make it clear it is a person backup. Maybe "Make sure your project has multiple people with access to the gemnasium dashboard."

Comment (wslack) [inline _pages/security/static-analysis.md]: done!

**Comment by wslack** [comment]
One question: Gemnaisum is still telling me I have a limited amount of time left on the free trial. Do I retain the devops@gsa.gov access after my free trial ends?

**Comment by wslack** [comment]
screen shot 2017-08-22 at 3 25 18 pm

**Comment by afeld** [comment]
@cmc333333 @erik-burgess Any idea?

**Comment by erik-burgess** [comment]
It looks fine on my screen. As we're billed annually, the next billing cycle is in October. 

Even though I invited @wslack to our org, I suspect that he's not in it.

**Comment by wslack** [comment]
I'm on the shared org, but it looks like they want me to buy something individually as well?

screen shot 2017-08-22 at 3 37 40 pm

**Comment by erik-burgess** [comment]
I don't know anything about that... This is what I see: 

screen shot 2017-08-22 at 3 39 43 pm

**Pull request merged**